packages/standard-minifier-js/plugin/minify-js.js

```diff
@@ -32,13 +32,110 @@ UglifyJSMinifier.prototype.processFilesForBundle = function (files, options) {
     }
   };
 
+  function maybeThrowMinifyErrorBySourceFile(error, file) {
+    var minifierErrorRegex = /\(line: (\d+), col: (\d+), pos: (\d+)\)/;
+    var parseError = minifierErrorRegex.exec(error.toString());
+
+    if (parseError) {
+      var lineErrorMessage = parseError[0];
+      var lineErrorLineNumber = parseError[1];
+
+      var parseErrorContentIndex = lineErrorLineNumber - 1;
+
+      // Unlikely, since we have a multi-line fixed header in this file.
+      if (parseErrorContentIndex < 0) {
+        return;
+      }
+
+      /*
+
+      What we're parsing looks like this:
+
+      /////////////////////////////////////////
+      //                                     //
+      // path/to/file.js                     //
+      //                                     //
+      /////////////////////////////////////////
+                                             // 1
+         var illegalECMAScript = true;       // 2
+                                             // 3
+      /////////////////////////////////////////
+
+      Btw, the above code is intentionally not newer ECMAScript so
+      we don't break ourselves.
+
+      */
+
+      var contents = file.getContentsAsString().split(/\n/);
+      var lineContent = contents[parseErrorContentIndex];
+
+      // Try to grab the line number, which sometimes doesn't exist on
+      // line, abnormally-long lines in a larger block.
+      var lineSrcLineParts = /^(.*?)(?:\s*\/\/ (\d+))?$/.exec(lineContent);
+
+      // The line didn't match at all?  Let's just not try.
+      if (!lineSrcLineParts) {
+        return;
+      }
+
+      var lineSrcLineContent = lineSrcLineParts[1];
+      var lineSrcLineNumber = lineSrcLineParts[2];
+
+      // Count backward from the failed line to find the filename.
+      for (var c = parseErrorContentIndex - 1; c >= 0; c--) {
+        var sourceLine = contents[c];
+
+        // If the line is a boatload of slashes, we're in the right place.
+        if (/^\/\/\/{6,}$/.test(sourceLine)) {
+
+          // If 4 lines back is the same exact line, we've found the framing.
+          if (contents[c - 4] === sourceLine) {
+
+            // So in that case, 2 lines back is the file path.
+            var parseErrorPath = contents[c - 2]
+              .substring(3)
+              .replace(/\s+\/\//, "")
+            ;
```

Comment:
Yeah, it's super dangly.  In my personal coding-life, I enjoy leaving room to add new in-between methods and having a cleaner `diff` by turning it into a makeshift curly bracket (to show what it matches up with).  That being said, this is a good call on your part and I've fixed it (as this style exists no-where in the code right now).